resources/translations/fi.edn

```diff
@@ -168,15 +168,15 @@
          :apply "Hae"
          :apply-for-bundle "Hae %1 aineistoa yhdessä "
          :header "%1 aineistoa korissa"
-         :intro [:div.intro [:p "Lisää resursseja koriin hakeaksesi niitä. Samalla hakemuksella haettavat resurssit ryhmitellään automaattisesti yhteen."]]
+         :intro [:div.intro [:p "Hae käyttöoikeuksia lisäämällä resursseja koriin. Useampaa oikeutta kerralla haettaessa keskenään samaa hakemusmenettelyä noudattavat resurssit yhdistyvät yhdeksi hakemukseksi."]]
```

Comment:
hakemusmenettely is not a REMS term, here too I like the previous version

Reply:
Previous version is much more easier to read and understand.

Reply:
Let's change it then: 

"Hae käyttöoikeuksia lisäämällä kielivaroja koriin. Useamman kielivaran käyttöoikeutta kerralla haettaessa keskenään samaa hakemusta käyttävät kielivarat yhdistyvät."

I have no idea where "resursseja" reappeared from here. It wasn't in the version I sent.

Reply:
I actually explained that (admittedly briefly and in parentheses) in the previous email I sent: these are the translations that are for the common use, so let's use "resource" here. After they're accepted, I'll make custom translations that are LBR-specific, in which "resurssi" has been changed to "kielivara" but are identical otherwise.

That is for technical reasons: LBR-specific translations are in our private repository, so having these discussions there would not work out so well.

Reply:
Opted to use Jaakko's proposal in this patch. Tero's proposal is a bit difficult to understand for me, most likely because of the participial phrase (lauseenvastike). Maybe it could be an actual clause instead?

Reply:
In the latter sentence, I guess (there's one in both)? We can change it to e.g. 

"Hae käyttöoikeuksia lisäämällä kielivaroja koriin. Samaa hakemusta käyttävät kielivarat yhdistyvät korissa yhdeksi hakemukseksi."

Reply:
And in Engllish: 

"Apply for access by adding resources to the cart. Resources using the same application are bundled together in the cart."

Reply:
Yes, in the latter sentence. I like that proposal. Done.